ceremonies/assignment/src/math.rs

```diff
@@ -17,18 +17,22 @@
 //! Math functions used in the ceremonies pallet.
 
 use encointer_primitives::{RandomNumberGenerator, RandomPermutation};
-use sp_runtime::traits::Hash;
-use sp_std::vec::Vec;
-
-pub fn checked_modulo(dividend: u64, divisor: u64) -> Option<u64> {
+use sp_runtime::traits::{Hash, Zero};
+use sp_std::{ops::Rem, vec::Vec};
+
+pub fn checked_modulo<T: Rem<T> + Rem<Output = T> + Zero + PartialEq>(
+	dividend: T,
```

Comment:
Nice that you added some generics here. Your trait bounds are a bit redundant though.

`T: Rem<T> + Rem<Output = T> == T: Rem<Output=T>` // `Rem<Output=T>` is a subset of `Rem<T>`